build.xml

```diff
@@ -13,11 +13,9 @@
         <os family="mac"/>
     </condition>
 
-    <echo>
-        ant.java.version: ${ant.java.version}
-        java.version:     ${java.version}
-        kernel:           ${kernel}
-    </echo>
+    <echo>ant.java.version: ${ant.java.version}
```

Comment:
Extremely minor, but I think the indentation got messed up there

Reply:
Unfortunately `ant` doesn't automatically detect and strip indentation. It takes and outputs this literally.
So, this "fixes" the output, which had leading whitespaces before.